usaspending_api/etl/management/commands/combine_transaction_search_chunks.py

```diff
@@ -0,0 +1,218 @@
+from usaspending_api.common.helpers.timing_helpers import ConsoleTimer as Timer
+from usaspending_api.common.matview_manager import DEFAULT_CHUNKED_MATIVEW_DIR
+
+logger = logging.getLogger("script")
+
+TABLE_NAME = "transaction_search"
+
+
+class Command(BaseCommand):
+
+    help = """
+    This script combines the chunked Universal Transaction Matviews and
+    combines them into a single table.
+    """
+
+    constraint_names = []
+
+    def add_arguments(self, parser):
+        parser.add_argument("--chunk-count", default=10, help="Number of chunked matviews to read from", type=int)
+        parser.add_argument("--index-concurrency", default=20, help="Concurrency limit for index creation", type=int)
+        parser.add_argument(
+            "--matview-dir",
+            type=Path,
+            help="Choose a non-default directory to store materialized view SQL files.",
+            default=DEFAULT_CHUNKED_MATIVEW_DIR,
+        )
+        parser.add_argument(
+            "--keep-old-data",
+            action="store_true",
+            default=False,
+            help="Indicates whether or not to drop old table at end of command",
+        )
+        parser.add_argument(
+            "--keep-matview-data",
+            action="store_true",
+            default=False,
+            help="Indicates whether or not to empty data from chunked matviews at the end of command",
+        )
+
+    def handle(self, *args, **options):
+        chunk_count = options["chunk_count"]
+        index_concurrency = options["index_concurrency"]
+        self.matview_dir = options["matview_dir"]
+
+        logger.info(f"Chunk Count: {chunk_count}")
+
+        # Constraints must be read first, so indexes created by pk/fk are not repeated
+        create_temp_constraints, rename_constraints = self.read_constraint_definitions()
+        create_temp_indexes, rename_indexes = self.read_index_definitions()
+
+        with Timer("Recreating table"):
+            self.recreate_matview()
+
+        with Timer("Inserting data into table"):
+            self.insert_matview_data(chunk_count)
+
+        with Timer("Creating table constraints"):
+            self.create_constraints(create_temp_constraints)
+
+        with Timer("Creating table indexes"):
+            self.create_indexes(create_temp_indexes, index_concurrency)
+
+        with Timer("Swapping Tables/Indexes"):
+            self.swap_matviews(rename_indexes, rename_constraints)
+
+        if not options["keep_old_data"]:
+            with Timer("Clearing old table"):
+                self.remove_old_data(chunk_count)
+
+        if not options["keep_matview_data"]:
+            with Timer("Emptying Matviews"):
+                self.empty_matviews()
+
+        with Timer("Granting Table Permissions"):
+            self.grant_matview_permissions()
+
+    def read_constraint_definitions(self):
+        indexes_sql = (self.matview_dir / "componentized" / "transaction_search__constraints.sql").read_text()
```

Comment:
If `transaction_search` is defined as a global var, should it replace the hardcoded strings?
```suggestion
        indexes_sql = (self.matview_dir / "componentized" / f"{TABLE_NAME}__constraints.sql").read_text()
```

Reply:
Added